File: Ask6/ask6.c
#include <stdio.h>
#include <math.h>

int main(){
    int numbers[250];
    int odd = 0; //Άρτιοι
    int even = 0; //Περιττοί
    for (int i = 1; i<=250;i++){
        printf("Type a number \n");
        scanf("%d",&numbers[i]);
        if (numbers[i] % 2 == 1){
            odd++;
        } else {
            even++;
        }
    }
    float p_odd = (odd/250)*100;
    float p_even = (even/250)*100;
    printf("Odd: %f%% \n Even: %f%%",p_odd, p_even);
    return 0;
}

File: Ask6/ask6.py
numbers = list(range(0,250))
odd = 0 #Περριτοί
even = 0 #Άρτιοι
for i in range (0,250):
    numbers[i] = int(input("Δώσε αριθμό \n >>> "))
    if numbers[i] % 2 == 0:
        odd+=1
    else:
        even+=1
p_odd = (odd/250)*100
p_even = (even/250)*100
print("Ποσοστό περιττών: ",p_odd,'%. \n Ποσοστό άρτιων: ',p_even,'%.')